apps/src/templates/studioHomepages/SetUpSections.jsx

```diff
@@ -12,7 +16,14 @@ class SetUpSections extends Component {
   };
 
   // Wrapped to avoid passing event args
-  beginEditingSection = () => this.props.beginEditingSection();
+  beginEditingSection = () => {
+    this.recordSectionSetupStartedEvent();
+    this.props.beginEditingSection();
+  };
+
+  recordSectionSetupStartedEvent = () => {
+    analyticsReporter.sendEvent(STARTED_EVENT, {});
```

Comment:
Is it worth making the payload an optional parameter for cases like this? (This could be updated in a follow-up PR if we think it is worth it.)

Reply:
might be. it only saves a couple characters, but might improve readability.

Reply:
Added a [followup task](https://codedotorg.atlassian.net/browse/TEACH-170), to not block this code from getting deployed any longer.